lib/classes/PluginManager.js

```diff
@@ -215,6 +220,24 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPathPatterns() {
+    return this.localPluginsPaths.map(({ resolvedPath, inputPath }) => {
+      const absoluteInputPath = path.resolve(this.serverless.serviceDir, inputPath);
+      if (
+        absoluteInputPath === resolvedPath ||
+        absoluteInputPath + path.extname(resolvedPath) === resolvedPath
+      ) {
+        return resolvedPath;
```

Comment:
I think resolved pattern should be in form of a path that's relative to service path (?), while here absolute path is returned